controllers/controllers/services/integration/cfservicebinding_integration_test.go

```diff
@@ -116,69 +147,95 @@ var _ = Describe("CFServiceBinding", func() {
 			).To(Succeed())
 		})
 
-		When("and the secret exists", func() {
-			It("eventually resolves the secretName and updates the CFServiceBinding status", func() {
-				Eventually(func() korifiv1alpha1.CFServiceBindingStatus {
-					updatedCFServiceBinding := new(korifiv1alpha1.CFServiceBinding)
-					Expect(
-						k8sClient.Get(context.Background(), client.ObjectKeyFromObject(cfServiceBinding), updatedCFServiceBinding),
-					).To(Succeed())
+		It("eventually resolves the secretName and updates the CFServiceBinding status", func() {
+			Eventually(func() korifiv1alpha1.CFServiceBindingStatus {
+				updatedCFServiceBinding := new(korifiv1alpha1.CFServiceBinding)
+				Expect(
```

Comment:
According the to Ginkgo documentation, when using expectations in `Eventually` one should follow the pattern of 

```
Eventually(func(g Gomega) {
    model, err := client.Find(1138)
    g.Expect(err).NotTo(HaveOccurred())
    g.Expect(model.Reticulate()).To(Succeed())
    g.Expect(model.IsReticulated()).To(BeTrue())
    g.Expect(model.Save()).To(Succeed())
}).Should(Succeed())
```

instead of calling `Expect` directly

Reply:
Yeah unfortunately our usage of Eventually seems to be pretty inconsistent across tests. Some places use the g Gomega arg and some don't. The docs seem to indicate we should be passing it in so I'll fix the tests here. I created this chore to go back and fix the other instances: https://github.com/cloudfoundry/korifi/issues/1240